test/coverage/datamodel/UUriSerializerTest.cpp

```diff
@@ -23,16 +23,167 @@ class TestFixture : public testing::Test {
 
 	// Run once per execution of the test application.
 	// Used for setup of all tests. Has access to this instance.
-	TestFixture() = default;
-	~TestFixture() = default;
+	TestUUriSerializaerFixture() = default;
+	~TestUUriSerializaerFixture() = default;
 
 	// Run once per execution of the test application.
 	// Used only for global setup outside of tests.
 	static void SetUpTestSuite() {}
 	static void TearDownTestSuite() {}
 };
 
-// TODO replace
-TEST_F(TestFixture, SomeTestName) {}
+std::shared_ptr<uprotocol::v1::UUri> buildValidTestURI(
+    const std::string& authority = "192.168.1.10") {
+	std::shared_ptr<uprotocol::v1::UUri> uri =
+	    std::make_shared<uprotocol::v1::UUri>();
```

Comment:
Either
```suggestion
	auto uri = std::make_shared<uprotocol::v1::UUri>();
```
or get rid of the pointer
```suggestion
	uprotocol::v1::UUri uri;
```

Reply:
I have removed shared pointer and used local UUri object.